### ISSUE TITLE
Suggestion: Manager specific Global Tabs setting

### ISSUE THREAD

**Issue by bossloper** [evolution-cms/evolution#663]
I find 'global tabs' useful as the prime admin for a site. However I would not want other managers to use them as they can be confusing for 'casual managers' and leave lots of resources open/locked if not careful.
Suggest adding a 'Use Global Tabs' option to manager configuration settings?

**Comment by pmfx** [comment]
But it is already in the settings. Also only Administrators can use it.

**Comment by bossloper** [comment]
I don't see it... or am I missing something (I am not referring to system configuration)? If I edit a manager and look in their 'user' tab I do not see a  'Use Global Tabs' option.

**Comment by pmfx** [comment]
Yup, I meant system configuration. Still, AFAIK it is only available for Administrators, so other users shouldn't be able to use it right now. 

I would like to have it as a new Role permissions setting along with tree nodes drag and drop.

**Comment by bossloper** [comment]
Ah, I think a misunderstanding. I am suggesting it should be a manager specific override for manager account settings. Then can decide if a particular manger has global tabs or not. So set somewhere here:
![untitled](https://user-images.githubusercontent.com/1744687/39425823-0af2375c-4c75-11e8-81a6-1b8254b1f98f.png)

**Comment by pmfx** [comment]
Yeah. That would make sense too.

**Comment by bossloper** [comment]
..."tree nodes drag and drop" role would be very useful!

**Comment by pmfx** [comment]
...and it was proposed earlier here #472
